src/tools/pub/publishing.md

```diff
@@ -36,10 +36,9 @@ are a few additional requirements for uploading a package:
   it's too large, consider splitting it into multiple packages, or cutting down
   on the number of included resources or examples.
 
-* Your package should only have hosted dependencies. Git dependencies are
-  allowed but strongly discouraged; not everyone using Dart has Git installed,
-  and Git dependencies don't support version resolution as well as hosted
-  dependencies do.
+* Your package should only have hosted dependencies from the default pub package
+  repository, and SDK dependencies (ie. `sdk: flutter`). This ensures that
```

Comment:
We avoid i.e. in docs, since people often mix up i.e. and e.g. (https://developers.google.com/style/word-list#letter-I). Also, we avoid unadorned “This”, since it can be unclear (https://developers.google.com/style/word-list#letter-T). How about:

Your package should depend only on hosted dependencies (from the default pub package server) and SDK dependencies (`sdk: flutter`). These restrictions ensure that…

Reply:
Much better! :)

I'll update the PR tomorrow, thanks.

Reply:
Cool, thanks!